Cloud common - security in the cloud closes #478 (#479)
bob-crutchley/courseware: topics/cloud/modules/cloud-security/README.md

These are the benefits that cloud security offers:
- **centralized security**
- **reduced costs**
- **reduced administration**
- **reliability**

### Centralised security

Moving your services to the cloud centralizes them, at the same time security is also centralized in one place.

On premises network would have numerous devices and endpoints, that still is the case once everything is in the cloud.

The bigger difference comes from it all being in one place, it becomes easier to analyse traffic and filtering, additionally monitoring of networking and policy updates becomes a more streamlined process.

A big benefit is that you can configure your services for disaster recovery as everything is in one place.

### Cost effectiveness

A big benefit of using cloud storage and security services is that there is no need to spend up-front cost for hardware.

This reduces capital expense at the same time reducing administrative overhead.

IT teams used to fight fight with security issues reactively (once they appear), where in the cloud it's more about proactive security (before they appear) that is running 24/7 without any need for human intervention.

### Reduced administration

> Once your services are with a reputable cloud provider, there will be no more manual security configurations or constant security updates.

Reviewer: ```suggestion
Once your services are with a reputable Cloud provider, there will be no more manual security configurations or constant security updates.
```

> These tasks are not only massive time drain but also adds a bit load on resources.

Reviewer: ```suggestion
These tasks are not only a massive time drain, but also places a big load on resources.
```

> This is taken away once you move to the cloud as all of the security is defined and controlled in one place.

Reviewer: ```suggestion
This is taken away once you move to the Cloud as all of the security is defined and controlled in one place.
```

### Reliability

> It wouldn't be an understatement to say that cloud services provide the best dependability out there.

Reviewer: ```suggestion
It wouldn't be an understatement to say that Cloud services provide the best dependability out there.
```

> The only caveat being that the right security measure need to be in place and configured correctly.

Reviewer: ```suggestion
The only caveat to this is that the right security measures need to be in place and configured correctly for it to be effective.
```

> Once that step is done, users can then safely have access to their data and application in the cloud.

Reviewer: ```suggestion
Once that step is done, users can then safely have access to their data and applications in the Cloud.
```

> The trend of moving to the cloud is there for a reason, what businesses are noticing is that moving their services to the cloud gives them many benefits.

Reviewer: ```suggestion
### Advantages

Businesses are noticing that moving their services to the Cloud gives them many benefits.
```

> These benefits include things like: *scaling*, *reduced costs*, *using agile systems*.

Reviewer: ```suggestion
These benefits include things like:
- *scaling*
- *reduced costs*
- *using agile systems*
```

> One thing businesses need to ensure is that they have complete confidence in the security that they implemented and that the systems are secure of threats like: *data theft*, *leakage*, *deletion* and *corruption*.

Reviewer: ```suggestion
One thing businesses need to ensure is that they have complete confidence in the security that they implemented.

This generally means that the systems are secure from threats like:
- *data theft*
- *leakage*
- *deletion*
- *corruption*
```

> It doesn't matter whether you're using public, private, hybrid cloud models, they are all susceptible to IT threads is mishandled.

Reviewer: ```suggestion
### Disadvantages

It doesn't matter whether you're using *Public*, *Private*, or *Hybrid* Cloud models, they are all susceptible to IT threads, mishandling, and human error.
```

> That is why businesses are cautious of moving their systems to the cloud, but the same threats can happen on on-premises infrastructure, hence everything boils down to how the security is implemented and managed.

Reviewer: ```suggestion
That is why businesses are cautious of moving their systems to the Cloud.

 However, given that the same threats can happen on on-premises infrastructure, everything boils down to how the security is implemented and managed.
```

## Tasks

> Task for this module is a research task.

Reviewer: ```suggestion
*This is a research-based Task.*
```

> **Investigate what are the most common security issues with: *Public*, *Private*, *Hybrid cloud***

Reviewer: ```suggestion
**Investigate what the most common security issues are for each of the : *Public*, *Private*, and *Hybrid* Cloud models.**
```
